params/config/sim_config.yaml

```diff
@@ -12,4 +12,7 @@ deterministic_size: True           # default: True
 # deterministic values rather than means of a random distribution
 # can be overridden by deterministic_arrival and deterministic_size
 #deterministic: True                # default: False
-run_duration: 100                  # default: 100
\ No newline at end of file
+run_duration: 100                  # default: 100
+
+# Optional: Trace file trace relative to the CWD. Please place trace in the working directory
+trace_path: params/traces/default_trace.csv
```

Comment:
How did you implement it? The trace overwrites the inter-arrival mean field? What if the trace doesn't start at time 0?

You should clarify with another comment